lib/views/home_content/map_feed/map_feed.dart

```diff
@@ -0,0 +1,71 @@
+import "package:flutter/material.dart";
+import "package:google_maps_flutter/google_maps_flutter.dart";
+import "package:hooks_riverpod/hooks_riverpod.dart";
+import "package:proxima/services/geolocation_service.dart";
+import "package:proxima/views/home_content/map_feed/maps/nearby_posts_map.dart";
+import "package:proxima/views/sort_option_widgets/feed_sort_option/map_sort_option_chips.dart";
+
+class MapFeed extends ConsumerStatefulWidget {
+  const MapFeed({super.key});
+
+  @override
+  MapFeedState createState() => MapFeedState();
+}
+
+class MapFeedState extends ConsumerState<MapFeed> {
```

Comment:
I think that we should rename "MapFeed" as I am not sure that the map is really a "feed"